docs/host-list.rst

```diff
@@ -2,90 +2,150 @@
-|`Time4VPS                           |Cheap VPSes, seemingly based in Lithuania.            |
-|<https://www.time4vps.eu/>`_        |                                                      |
-+------------------------------------+------------------------------------------------------+
-|`Linode <https://www.linode.com/>`_ |More cheap VPSes!                                     |
-+------------------------------------+------------------------------------------------------+
-|`Vultr <https://www.vultr.com/>`_   |US-based, DigitalOcean-like.                          |
-+------------------------------------+------------------------------------------------------+
-
-------
-Others
-------
-
-+-------------------------------------+-----------------------------------------------------+
-|Link                                 |                                                     |
-+=====================================+=====================================================+
-|`AWS <https://aws.amazon.com/>`_     |Amazon Web Services. Free for a year (with certain   |
-|                                     |limits), but very pricey after that.                 |
-+-------------------------------------+-----------------------------------------------------+
-|`Google Cloud                        |Same as AWS, but it's Google.                        |
-|<https://cloud.google.com/compute/>`_|                                                     |
-+-------------------------------------+-----------------------------------------------------+
-|`Microsoft Azure                     |Same as AWS, but it's Microsoft.                     |
-|<https://azure.microsoft.com>`_      |                                                     |
-+-------------------------------------+-----------------------------------------------------+
-|`Oracle Cloud                        |Same as AWS, but it's Oracle.                        |
-|<https://oracle.com/cloud/>`_        |                                                     |
-+-------------------------------------+-----------------------------------------------------+
-|`LowEndBox <http://lowendbox.com/>`_ |A curator for lower specced servers.                 |
-+-------------------------------------+-----------------------------------------------------+
 
 ------------
-Self-hosting
+Self Hosting
 ------------
 
-You can always self-host on your own hardware.
-A Raspberry Pi 3 will be more than sufficient for small to medium sized bots.
+| It's possible to self host Red with one's own hardware. A Raspberry Pi 3 
+  will have enough computing capacity to handle a small to medium sized bot. 
+  One also has the option of building a computer for this purpose or purchasing 
+  a  rack server as any modern hardware should work without issues. But keep in
+  mind this option can leaves one vulnerable to factors outside their control
+  such as power outages.
```

Comment:
```suggestion
  You can also host on your own computer or rack server. Any modern hardware 
  should work without issues. However, this option leaves you responsible for
  keeping the bot online by paying for electricity costs and dealing with power outages.
```
I rewrote this section a bit, changed the focus to what I think is the major drawbacks to selfhosting. If you want to rewrite it again in a different way, feel free.

Reply:
Better than anything i could come up with